Source/Samples/Sections/Widgets/EntrySection.cs

```diff
@@ -66,5 +69,100 @@ public EntrySection()
             return ("Invisible text entry:", entry);
         }
 
+        public (string, Widget) CreateSearchEntry()
+        {
+            var entry = new Entry();
+            entry.PlaceholderText = "Search";
+            entry.SetIconFromIconName(EntryIconPosition.Primary, "edit-find");
```

Comment:
Please use symbolic icons when you can, otherwise the icons look bad with the dark theme ("edit-find" -> "edit-find-symbolic")